src/treq/test/test_auth.py

```diff
@@ -130,6 +131,28 @@ def test_add_basic_auth_bytes(self):
             Headers({b'Authorization': [b'Basic AQ//Ov/wAQ==']}),
         )
 
+    def test_add_digest_auth(self):
```

Comment:
Can you add a docstring for this test explaining what it's testing specifically?

Reply:
added